tests/examples/variables.tf

```diff
@@ -62,6 +62,10 @@ variable "region" {
   default = "europe-west8"
 }
 
+variable "region_1" {
+  default = "europe-west9"
+}
+
```

Comment:
What about using the FAST convention?

```hcl
regions = {
  primary     = "europe-west1"
  secondary = "europe-west3"
}
```

Reply:
Since its a list, thought there can be morethan two regions as well. Assumed that renaming as primary and secondary might give an impression to users that only 2 regions can be used for replication.

Reply:
> Let's call it `region_a` and `region_b`. Also, we need to add those to setup module. (with assumption that `var.region_a == var.region` and `var.region_a != var.region_b`)

Sure, let me try this.

Reply:
> What about using the FAST convention?
> 
> ```terraform
> regions = {
>   primary   = "europe-west1"
>   secondary = "europe-west3"
> }
> ```

Thanks Ludo, 

Since its a list, thought there can be morethan two regions as well. Assumed that renaming as primary and secondary might give an impression to users that only 2 regions can be used for replication.